### PR TITLE
Update makefile and clean up README

### PR DESCRIPTION
**Pull request by Ricardo-Osorio**
This PR started with a completely different goal in mind but it has been dropped since then so I discarded all changes except for the ones still relevant and not directly related to that past goal.

Clean the README a little bit:
- specified the language on the code blocks
- simplified some titles and phrases
- fixed typos
- moved some inline code to code blocks where this change made it more readable
- remove the outputs of the cli
  - it's one more place where we need to keep it up to date with every change to the codebase
  - makes the README too long
  - does not add value

Update makefile to include build args on the run cmd the same way these are included in the build cmd

Run github actions only on push. Avoids running twice on many occasions (push/PR)

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -246,28 +114,30 @@ spec:
...
+## Enable TLS
 
-## TLS Enablement
+### Install ETCD and StorageOS with TLS enabled
 
-### Install ETCD and StorageOS With TLS Enabled
-
-`kubectl storageos install --include-etcd --etcd-tls-enabled`
+```bash
+kubectl-storageos install --include-etcd --etcd-tls-enabled
```
Comment (mhmxs) [inline README.MD:124]: I think `kubectl storageos` is the right syntax

File: README.MD [hunk 2]
```diff
@@ -246,28 +114,30 @@ spec:
...
 
 In order for StorageOS to connect to ETCD over HTTPS, a k8s secret must exist in the `storageos-cluster-namespace` with valid ETCD credentials.
 This secret can be created like so:
 
 ```lang-none
-kubectl create secret generic <etcd-secret-name> -n <storageos-cluster-namespace> \
-                --from-file=etcd-client-ca.crt=path/to/ca.crt \
-                --from-file=etcd-client.crt=path/to/tls.crt \
-                --from-file=etcd-client.key=path/to/tls.key
+kubectl-create secret generic <etcd-secret-name> -n <storageos-cluster-namespace> \
```
Comment (mhmxs) [inline README.MD:137]: `kubectl-create`? I have never used that command.

File: README.MD (hunk 1 above)
Comment (Ricardo-Osorio) [reply README.MD]: Reverted

File: README.MD (hunk 2 above)
Comment (Ricardo-Osorio) [reply README.MD]: Me neither :D

**Review by mhmxs — APPROVED** [review]

**Pull request merged**